### DISCUSSION TITLE
CSS selector for "search in tab"

### DISCUSSION THREAD

**Discussion by jra66** [xiaoyifang/goldendict-ng#1686] [Q&A] [answered]
![Bild 001](https://github.com/user-attachments/assets/848aa8c3-0e95-4d55-8288-6d0a2b554cbd)

Hi,

in the past I have been using original GoldenDict with a qt-style.css to theme the program like [Solarized Light](https://ethanschoonover.com/solarized). Unfortunately I am not able to style "search in tab" (see image), which remains white. Can someone please provide me a CSS selector to style it?

Kind regards,

Jörg

**Comment by shenlebantongying** [comment, marked as answer]
The class of the search panel is `SearchPanel`.

Sadly, it appears that Qt stylesheet doesn't work with derived classes, some changes are needed to directly modify it. This can be considered as a bug.

However, changing the background of the entire app still works.

```css
QWidget {
	background-color: #fdf6e3;
}
```

![image](https://github.com/user-attachments/assets/decd00b6-5f4b-4d8b-ad6f-559ea611649c)

**Reply by jra66** [reply]
Thanks s lot for your help,

Jörg
